scripts/env/jlse.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh -e
+
+module use /soft/modulefiles
+export PROJ=/vast/projects/celeritas
+export SPACKROOT=/vast/projects/celeritas/spack
+. $SPACKROOT/share/spack/setup-env.sh
+
+# Interactive job on an MI30
+alias mi300x="qsub -I -t 60 -n 1 -q gpu_amd_mi300x"
+
+# Interactive job on an H100
+alias h100="qsub -I -t 60 -n 1 -q gpu_h100"
```

Comment:
Maybe write as a function:
```c++
qsub-gpu() {
    # Usage: qsub-gpu [mi300x|h100|...]
    case "$1" in
        mi300x) queue=gpu_amd_$1 ;;
        *) queue=gpu_$1 ;;
    esac
    exec qsub -q "$queue" "$@"
}
```
and you can `alias mi300x=qsub-gpu mi300x -I -t 60 -n 1` in your bashrc? 

It's great to have these GPU names here as documentation but everyone's going to want their own aliases.

Also maybe these should be in an `else` block since the jobs should be submitted only from the host `jlse*` node?